daemon/mgr/container_utils.go

```diff
@@ -379,3 +392,20 @@ func amendContainerSettings(config *types.ContainerConfig, hostConfig *types.Hos
 		r.MemorySwap = 2 * r.Memory
 	}
 }
+
+// isCFQScheduler checks if dev slice is all cfq io scheduler
+func isCFQScheduler(dev []string) bool {
+	for _, d := range dev {
+		index := strings.LastIndex(d, "/")
+		raw, err := ioutil.ReadFile(fmt.Sprintf("/sys/block/%s/queue/scheduler", d[index+1:]))
```

Comment:
Can we mock this line of code? @shaloulcy 
Maybe you can give us more input of this.

Reply:
sysDirTemp="/sys/block/%s/queue/scheduler"

func isCFQScheduler(sysDirTemp string, dev []string) bool{
        for _, d := range dev {
              index := strings.LastIndex(d, "/")
              raw, err := ioutil.ReadFile(fmt.Sprintf(sysDirTemp,d[index+1:])
       }
}

Reply:
Sorry, but not think the example is good, generally, we add unit test for this type of functions will defines a path, and we will mock a file and make it a function parameter. For this, path is something we need to splice from parameter, if we do like example, it just make code ugly.
So other way, WDYT, @shaloulcy